app/notifications/reimbursement_complete_notification.rb

```diff
@@ -0,0 +1,19 @@
+class ReimbursementCompleteNotification < BaseNotification
+  deliver_by :database
+
+  param :case_contact
+
+  def title
+    "Reimbursement Approved"
+  end
+
+  def message
+    case_contact = params[:case_contact]
+    # Questions about logic for payment rate
+    "Volunteer #{case_contact.creator.display_name}'s request for reimbursement for #{case_contact.miles_driven}mi on #{case_contact.occurred_at_display} has been processed and is en route."
```

Comment:
Does this need a space between `#{case_contact.miles_driven}` and `mi`?

Reply:
The original ticket example had no space.